automation/build.sbt

```diff
@@ -58,7 +58,7 @@ Test / logBuffered := false
   *  Warning: can't set too high (set at 10 would crashes OS)
   *  This is not number of threads in each JVM. That would be up to sbt.
   */
-Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 4))
+Global / concurrentRestrictions := Seq(Tags.limit(Tags.ForkedTestGroup, 6))
```

Comment:
Is this change related to reorganizing method tests?

Reply:
yes and no. number of JVMs are upped to see whether there are undetected Methods tests interference.